code/defines/obj/weapon.dm

```diff
@@ -81,11 +81,78 @@
 	icon_state = "cane"
 	item_state = "stick"
 	flags = CONDUCT
-	force = 10
+	force = 0

```

Comment:
Is there a particular reason for this? You're already overriding attack(), and there are other pieces of code which might expect a saner value.

Reply:
For example, this will make canes do no throw damage, I am sure.